.github/workflows/_ci.yaml

```diff
@@ -0,0 +1,205 @@
+    secrets: inherit
+
+  build-t5x:
+    needs: [metadata, build-jax]
+    if: inputs.ARCHITECTURE == 'amd64'  # T5X arm64 build is wip in PR 252
+    uses: ./.github/workflows/_build_t5x.yaml
+    with:
+      ARCHITECTURE: amd64
+      BUILD_DATE: ${{ needs.metadata.outputs.BUILD_DATE }}
+      BASE_IMAGE: ${{ needs.build-jax.outputs.DOCKER_TAG_MEALKIT }}
+      REPO_T5X: ${{ needs.metadata.outputs.REPO_T5X }}
+      REF_T5X: ${{ needs.metadata.outputs.REF_T5X }}
+    secrets: inherit
+
+  build-pax:
+    needs: [metadata, build-jax]
+    uses: ./.github/workflows/_build_pax.yaml
+    with:
+      ARCHITECTURE: ${{ inputs.ARCHITECTURE }}
+      BUILD_DATE: ${{ needs.metadata.outputs.BUILD_DATE }}
+      BASE_IMAGE: ${{ needs.build-jax.outputs.DOCKER_TAG_MEALKIT }}
+      REPO_PAXML: ${{ needs.metadata.outputs.REPO_PAXML }}
+      REF_PAXML: ${{ needs.metadata.outputs.REF_PAXML }}
+      REPO_PRAXIS: ${{ needs.metadata.outputs.REPO_PRAXIS }}
+      REF_PRAXIS: ${{ needs.metadata.outputs.REF_PRAXIS }}
+    secrets: inherit
+
+  build-rosetta-t5x:
+    uses: ./.github/workflows/_build_rosetta.yaml
+    needs: [metadata, build-t5x]
+    with:
+      ARCHITECTURE: ${{ inputs.ARCHITECTURE }}
+      BUILD_DATE: ${{ needs.metadata.outputs.BUILD_DATE }}
+      BASE_IMAGE: ${{ needs.build-t5x.outputs.DOCKER_TAG_MEALKIT }}
+      BASE_LIBRARY: t5x
+    secrets: inherit
+
+  build-rosetta-pax:
+    uses: ./.github/workflows/_build_rosetta.yaml
+    needs: [metadata, build-pax]
+    with:
+      ARCHITECTURE: ${{ inputs.ARCHITECTURE }}
+      BUILD_DATE: ${{ needs.metadata.outputs.BUILD_DATE }}
+      BASE_IMAGE: ${{ needs.build-pax.outputs.DOCKER_TAG_MEALKIT }}
+      BASE_LIBRARY: pax
+    secrets: inherit
+
+  test-distribution:
+    needs: metadata
+    uses: ./.github/workflows/_test_distribution.yaml
+    secrets: inherit
+
+  test-jax:
+    needs: build-jax
+    if: inputs.ARCHITECTURE == 'amd64'  # arm64 runners n/a
+    uses: ./.github/workflows/_test_jax.yaml
+    with:
+      JAX_IMAGE: ${{ needs.build-jax.outputs.DOCKER_TAG_FINAL }}
+    secrets: inherit
+
+  test-te:
+    needs: build-pax
+    if: inputs.ARCHITECTURE == 'amd64'  # arm64 runners n/a
+    uses: ./.github/workflows/_test_te.yaml
+    with:
+      TE_IMAGE: ${{ needs.build-pax.outputs.DOCKER_TAG_FINAL }}
+    secrets: inherit
+
+  test-t5x:
+    needs: build-t5x
+    if: inputs.ARCHITECTURE == 'amd64'  # arm64 runners n/a
+    uses: ./.github/workflows/_test_t5x.yaml
+    with:
+      T5X_IMAGE: ${{ needs.build-t5x.outputs.DOCKER_TAG_FINAL }}
+    secrets: inherit
+
+  test-pax:
+    needs: build-pax
+    if: inputs.ARCHITECTURE == 'amd64'  # arm64 runners n/a
+    uses: ./.github/workflows/_test_pax.yaml
+    with:
+      PAX_IMAGE: ${{ needs.build-pax.outputs.DOCKER_TAG_FINAL }}
+    secrets: inherit
+
+  test-vit:
+    needs: build-rosetta-t5x
+    uses: ./.github/workflows/_test_vit.yaml
+    with:
+      ROSETTA_T5X_IMAGE: ${{ needs.build-rosetta-t5x.outputs.DOCKER_TAGS }}
```

Comment:
Should this be `needs.build-rosetta-t5x.outputs.DOCKER_TAG_FINAL`?

Reply:
Done